### ISSUE TITLE
Ignition source can use the S3 schema, but docs say otherwise

### ISSUE THREAD

**Issue by Sodki** [coreos/bugs#2067]
# Issue Report #

Bug

### Container Linux Version ###

```
$ cat /etc/os-release
NAME="Container Linux by CoreOS"
ID=coreos
VERSION=1465.2.0
VERSION_ID=1465.2.0
BUILD_ID=2017-07-19-0414
PRETTY_NAME="Container Linux by CoreOS 1465.2.0 (Ladybug)"
ANSI_COLOR="38;5;75"
HOME_URL="https://coreos.com/"
BUG_REPORT_URL="https://issues.coreos.com"
COREOS_BOARD="amd64-usr"
```

### Environment ###

AWS

### Expected Behavior ###

Ignition documentation for 2.1+ states that for `ignition.config.append.source` and `ignition.config.replace.source`: `Supported schemes are http, https, and tftp`. This doesn't seem to be right, because I'm using it with `s3` successfully and I don't see anything in the code that might limit the supported schemes for these keys. Compare it, for example, with `storage.files.contents.source`, which states `Supported schemes are http, https, tftp, s3, and data`.

### Actual Behavior ###

`s3` is actually a supported schema.

**Comment by bgilbert** [comment]
The PXE and iPXE OS docs should also be updated.

**Comment by bgilbert** [comment]
This is fixed in Ignition 0.17.2, and the Container Linux docs will be fixed in 1492.0.0.  Thanks for the report.
